docs/en_US/Tutorial/HowToLaunchFromPython.rst

```diff
@@ -0,0 +1,103 @@
+**How to Launch from Python**
+=============================
+
+Overview
+--------
+This is a new way to launch experiments since ``nni v2.0``. Before, you need to configure your experiment in a yaml configuration file and launch the experiment through ``nnictl`` command. Now, you can also configure and run your experiment directly in the python file. If you are familiar with python programming, this will undoubtedly bring you more convenience.
+
+How to Use
+----------
+After successfully installing ``nni``, you can write your own experimental configuration and run. You can even use python script to launch multiple experiments as you like! The following three steps are necessary.
+
+..
+
+    Step 1 - Initialize a tuner you want to use
+
+
+.. code-block:: python
+
+    from nni.algorithms.hpo.foo_tuner import FooTuner
+    foo_tuner = FooTuner()
+
+Very simple, you have successfully initialized a ``FooTuner`` instance called ``foo_tuner``.
+
+See all real `builtin tuners <../builtin_tuner.rst>`__ supported in NNI.
+
+..
+
+    Step 2 - Initialize a experiment instance and configure it
+
+.. code-block:: python
+
+    experiment = Experiment(tuner=foo_tuner, training_service='local')
+
+Now, you have a ``Experiment`` instance with ``foo_tuner`` you have initialized in the previous step, and this experiment will launch trials on your local machine due to ``training_service='local'``.
+
+See all `training services <../training_services.rst>`__ supported in NNI.
+
+.. code-block:: python
+
+    experiment.config.experiment_name = 'test'
```

Comment:
better to make this part more real, for example, setting up a real local experiment

Reply:
fix it